std/container/binaryheap.d

```diff
@@ -516,11 +516,11 @@ BinaryHeap!(Store, less) heapify(alias less = "a < b", Store)(Store s,
     static struct StructWithoutDup
     {
         int[] a;
-        @disable StructWithoutDup dup()
-        {
-            StructWithoutDup d;
-            return d;
-        }
+        @disable StructWithoutDup dup();
+        //{
+        //    StructWithoutDup d;
+        //    return d;
+        //}
```

Comment:
No need to comment out a useless function body, just delete it.

Reply:
Done. Thank you for the input!